simbad/mr/phaser_mr.py

```diff
@@ -283,11 +284,11 @@ def run(self):
             i.setJOBS(1)
             i.setREFL_DATA(r.getREFL_DATA())
             i.setROOT("phaser_mr_output")
-            i.addENSE_PDB_RMS("PDB", pdbin, 1.2)
+            i.addENSE_PDB_ID("PDB", pdbin, 0.7)
```

Comment:
Hard coded sequence id? Maybe dump that in a variable as kwarg to this function. At least allow modification from outside